@opencensus/exporter-azure missing repository field, can't be published with wombot

`@opencensus/exporter-azure` was not published in this last release because it is missing the repository field in package.json, causing publishing through wombot to complain:

```
npm ERR! code E400
npm ERR! 400 Bad Request - PUT https://wombat-dressing-room.appspot.com/@opencensus%2fexporter-azure -
npm ERR!   ===============================
npm ERR!   Publish service error
npm ERR!   -------------------------------
npm ERR!   in order to publish the latest version must have a repository aabmass can access.
npm ERR!   ===============================
npm ERR!
```

I believe adding a repository field should fix this

--- a/packages/opencensus-example-automatic-tracing/package.json
+++ b/packages/opencensus-example-automatic-tracing/package.json
@@ -4,6 +4,7 @@
   "version": "0.1.0",
   "private": true,
   "main": "zipkin.js",
+  "repository": "census-instrumentation/opencensus-node",
   "dependencies": {
     "@opencensus/exporter-instana": "^0.1.0",
     "@opencensus/exporter-stackdriver": "^0.1.0",

--- a/packages/opencensus-exporter-azure/package.json
+++ b/packages/opencensus-exporter-azure/package.json
@@ -4,6 +4,7 @@
   "description": "OpenCensus Azure Exporter allows the user to send stats and traces collected with OpenCensus Node.js to Azure Monitor.",
   "type": "module",
   "main": "index.ts",
+  "repository": "census-instrumentation/opencensus-node",
   "scripts": {
     "test": "nyc mocha -r ts-node/register test/**/*.ts",
     "compile": "tsc -p .",

